packages/stylable.io/docs/references/imports.md

```diff
@@ -78,15 +144,34 @@ The values `gridMixin` and `tooltipMixin` are imported from the local JavaScript
 import { gridMixin, tooltipMixin as tooltip } from "./my-mixins";
 ```
 
-## Import stylesheet
+## Import keyframes
+
+In Stylable, both class names and keyframes undergo namespacing to avoid collision. However, despite the two being global, they do not share a namespace in CSS - this means that you can have both a class name and a keyframe with the same name.
+
+Due to this, when importing keyframes from another stylesheet, a special `keyframes()` directive needs to be used.
+
+```css
+/* comp.st.css - new syntax */
+@st-import [keyframes(slideX, slideY)] from './keyframes.st.css';
+```
+
+```css
+/* comp.st.css - old syntax */
+:import {
+    -st-from: "./my-mixins";
+    -st-named: keyframes(slideX, slideY);
+}
+```
+
+You can read more about keyframes behavior [here](./keyframes.md).
 
-When importing another stylesheet, the default import represents the root of the stylesheet and is generally treated as a component, and named imports represent vars and classes.
 
-## Usage
+## Importing specific symbols
```

Comment:
heading is a bit misleading because the content links might contain `import` usages, but it doesn't specifically link to them.